README.md

```diff
@@ -1,97 +1,330 @@
 
-## Introduction 
-This playbook offers guidance and templates to improve the applicant experience of providing income in SNAP applications and renewals. In this playbook, you’ll find: 
-* Recommended practices 
-* Common terms and plain language definitions 
-* Example language and visual templates for income reporting 
+**_This playbook is \*\*NOT\*\* a source of authoritative, legal, or regulatory guidance and has not been officially endorsed by the Center for Medicaid and CHIP Services (CMCS) or the Food and Nutrition Service (FNS). It is advisory only, and should be adapted appropriately for each state and scenario. Ultimately, it is the responsibility of state Medicaid and Supplemental Nutrition Assistance Program (SNAP) officials to ensure that implementation of any project is compliant with federal Medicaid and SNAP statute and regulations._**
 
-## Who is this guide for? 
-This guide is for: 
-* SNAP program administrators and their staff 
-* Other state, county, and local officials who support the SNAP eligibility and enrollment process 
-* Vendors and contractors working with agencies to deliver SNAP benefits to applicants and/or clients 
+**_Refer to CMCS' [website](https://www.medicaid.gov/medicaid/index.html) and FNS' [website](https://www.fns.usda.gov/snap/supplemental-nutrition-assistance-program) for up-to-date official guidance._**
 
-This guide is not intended to be used by SNAP clients. 
+### Table of Contents
 
-## How can I use the information in this guide? 
-* Identifying changes to make to your application/renewal form
-* Identifying areas of your application/renewal form to test with clients
+* [Introduction](#introduction)
+* [What is plain language?](#what-is-plain-language)
+* [Impact of plain language implementations](#impact-of-plain-language-implementations)
+* [Recommended best practices](#recommended-practices)
+* [Lexicon: Common terms and definitions](#lexicon-common-terms-and-definitions)
+* [Self-attestations and “no income” situations](#self-attestations-and-no-income-situations)
+* [Additional improvement areas](#additional-improvement-areas)
+* [Considerations for cross-program contexts](#considerations-for-cross-program-contexts)
+* [Closing](#closing)
+* [Provide Feedback](#provide-feedback)
+
+## Introduction
+
+Eligible people, who have the most need for healthcare, cash assistance, and food assistance are struggling to receive or renew benefits. This struggle is in part due to the burden of income verification. The burden of proof falls on them, not the government service.
+
+<div>
+The Income Reporting Playbook offers guidance and templates to improve the <div class="tooltip">client<span class="tooltiptext">“Client” refers to people applying for or renewing a benefit.</span></div> experience of providing income during benefit applications and renewals.
```

Comment:
These tooltips are cool and helpful!

Reply:
BUT the text in them is not centered... looks like we need more margin/padding on the left.

<img width="296" alt="image" src="https://github.com/navapbc/FFS-IncomeReportingToolkit/assets/776667/e5a1a0a0-0f00-45c2-86d1-5c2bfb567ea7">